api/src/main/java/com/foro/api/topic/ReplyDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.foro.api.topic;
+
+import jakarta.persistence.*;
+import lombok.*;
```

Comment:
Let's try not to use * imports but specific imports, Intellij by default uses * imports but this feature can be disabled.